src/main/java/com/pixonic/ctop/Main.java

```diff
@@ -8,26 +8,42 @@
 import javax.management.remote.JMXConnector;
 import javax.management.remote.JMXConnectorFactory;
 import javax.management.remote.JMXServiceURL;
+import java.util.HashMap;
 
 public class Main {
 
     public static void main(String[] args) throws Exception {
         if (args.length == 0) {
-            System.out.println("Usage: java -jar ctop.jar <host:jmx_port> <key_space> [interval_sec(default: 10)]");
+            System.out.println("Usage: java -jar ctop.jar <host:jmx_port> <key_space> [interval_sec(default: 10)] [jmxUsername:jmxPassword]");
             return;
         }
 
         String hostAndPort = args[0];
         String keySpace = args[1];
+
         int interval = 10;
         if (args.length > 2) {
             interval = Integer.parseInt(args[2]);
         }
+        
```

Comment:
If we pass arguments as “host:20000 key_space user:password”, NumberFormatException will be thrown.